miri-script/src/main.rs

```diff
@@ -144,10 +168,10 @@ impl Command {
         }
 
         match self {
-            Self::Install { flags }
-            | Self::Build { flags }
-            | Self::Check { flags }
-            | Self::Doc { flags }
+            Self::Install { flags, .. }
+            | Self::Build { flags, .. }
+            | Self::Check { flags, .. }
+            | Self::Doc { flags, .. }
             | Self::Fmt { flags }
             | Self::Toolchain { flags }
             | Self::Clippy { flags }
```

Comment:
I think it'd make sense to also support this with clippy.

Reply:
I used this to check which cargo commands can take --features but cargo clippy instead had a line pointing to `check`. Fixed now.
```sh
for i in install build check clippy test run doc fmt bench toolchain; do echo $i; cargo $i --help | grep features; done
 ```